packages/backend/src/db/house/house.schema.ts

```diff
@@ -19,6 +19,9 @@ export class House {
 
   @Prop({ type: Types.ObjectId, ref: 'User' })
   owner: Types.ObjectId;
+
+  @Prop()
```

Comment:
there is probably a way to tell mongoose that it is an array of ids, similar to line 20 above.